index.bs

```diff
@@ -169,11 +207,71 @@ This allows developers to detect support for paint timing for a particular [=bro
 Processing model {#sec-processing-model}
 ========================================
 
+Recording paint timing {#sec-recording-paint-timing}
+--------------------------------------------------------
+
+<h4 id="sec-modifications-dom">Modifications to the DOM specification</h4>
+
+Each {{Element}} has an <dfn>associated image request</dfn> which is initially null.
+When the processing model for an {{Element}} <em>element</em> of type {{HTMLImageElement}}, {{SVGImageElement}}, or {{HTMLVideoElement}} creates a new image resource (e.g., to be displayed as an image or poster image), <em>element</em>'s <a>associated image request</a> is set to the <a>image request</a> of the created image resource.
+
+Note: Every image resource that is obtained from a URL whose <a spec=url>scheme</a> is equal to "data" has an associated <a>image request</a> which is not fetched but still needs to be loaded.
+This request can be the <a>associated image request</a> of an {{Element}}.
+
+Note: The current language is vague since it does not point to specific algorithms.
+This can be made more rigorous when the corresponding processing models have a more unified processing model.
+
+When an {{Element}} <em>element</em>'s <a>associated image request</a> has become <a>completely available</a>, run the algorithm to <a>process an image that finished loading</a> passing in <em>element</em> and its <a>associated image request</a> as inputs.
+
+<h4 id="sec-modifications-CSS">Modifications to the CSS specification</h4>
+
+When the user agent is executing the <a>painting order</a>, it must populate the <a>set of owned text nodes</a> of the painted {{Element|Elements}} so that the following is true:
+
+<div algorithm="text aggregation">
+    * If a {{Text}} object |text| will not be painted due to the font face being in its <a>font block period</a>, then it is not <a for="set">appended</a> to the <a>set of owned text nodes</a> of any {{Element}}.
+    * Otherwise, |text| is <a for="set">appended</a> to the <a>set of owned text nodes</a> of the {{Element}} which determines the <a>containing block</a> of |text|.
+</div>
+
+NOTE: A user agent might want to use a stack to efficiently compute the <a>set of owned text nodes</a> while implementing the <a>painting order</a>.
+
+Every {{Element}} has a list of <dfn>associated background image requests</dfn> which is initially an empty array.
```

Comment:
Only backgrounds or also any CSS image? (border-image etc)

Reply:
Currently (per existing spec) just backgrounds. But see https://github.com/w3c/largest-contentful-paint/issues/107 for the border-image discussion